python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -1022,6 +1031,18 @@ def errFunc(p, x, y):
             # This algorithm was initially developed by Seth Digel for
             # the EO Testing pipeline.
 
+            if self.config.scaleMaxSignalInitialPtcOutlierFit:
+                approxGain = np.nanmedian(meanVecOriginal/varVecOriginal)
+                maxADUInitialPtcOutlierFit = self.config.maxSignalInitialPtcOutlierFit/approxGain
+                self.log.info(
```

Comment:
We were concerned in another ticket about too much info in the logs, especially per amp. Will it be OK in this case? Should we change the level to "debug"?

Reply:
I did think about this, and could make it debug.  But it's not that much.  The difference is that this is a log that is done per amp in a PTC fit (16 messages per hundreds of flat pairs) and not a log per amp per ISR (16 messages times hundreds of flat pairs).